docs/source/using/basic-usage.adoc

```diff
@@ -13,11 +14,47 @@ toc::[]
 
 When you use {project}, you interact with two components:
 
-- a virtual machine (VM) created by {project}
+- the {project} virtual machine (VM)
 - the OpenShift cluster provisioned by {project} within the VM
 
-The following sections contain information about managing the {project} VM.
-For details about using {project} to manage your local OpenShift cluster, see the xref:../openshift/index.adoc#[Interacting with OpenShift] section.
+The xref:../using/managing-minishift.adoc#diagram-minishift-architecture[{project} architecture] diagram outlines these components.
+The `minishift` binary, placed somewhere on `PATH`, is used to xref:../using/basic-usage.adoc#minishift-start-overview[`start`], xref:../using/basic-usage.adoc#minishift-stop-overview[`stop`] and xref:../using/basic-usage.adoc#minishift-delete-overview[`delete`] the {project} VM.
+It also interacts with the Docker daemon running within the VM, for example via xref:../using/docker-daemon.adoc#reusing-docker-daemon[`docker-env`].
```

Comment:
This link also needs fix. I do not think we have `using/docker-daemon.adoc#reusing-docker-daemon` anymore.